samples/SkijaInjectSample/src/main/kotlin/SkijaInjectSample/App.kt

```diff
@@ -15,6 +15,7 @@ import kotlin.math.cos
 import kotlin.math.sin
 
 fun main(args: Array<String>) {
+    System.setProperty("skiko.renderApi", "OPENGL")
```

Comment:
To be removed?

Reply:
Yes, I forgot to delete

Reply:
Checked on linux (ubuntu 20.04) - works fine. Now this PL is ready for merging, if you don't mind.